uiServices/services.go

```diff
@@ -0,0 +1,57 @@
+package services
+
+import (
+	"encoding/json"
+	"net/http"
+	"bytes"
+	"github.com/Sirupsen/logrus"
+	"github.com/xtracdev/automated-perf-test/perfTestUtils"
+	"os"
+)
+
+
+func configsHandler(rw http.ResponseWriter, req *http.Request) {
+
+	configPathDir := req.Header.Get("configPathDir")
+
+	config := perfTestUtils.Config{}
+	buf := new(bytes.Buffer)
+	buf.ReadFrom(req.Body)
+	defer req.Body.Close()
+
+	err := json.Unmarshal(buf.Bytes(), &config)
+	if err != nil {
+		logrus.Error("Failed to unmarshall json body", err)
+		rw.WriteHeader(http.StatusBadRequest)
+		return
+	}
+
+	if FilePathExist(configPathDir)== false{
+		logrus.Error("File path does not exist", err)
+		rw.WriteHeader(http.StatusNotFound)
+		return
+
+	}
+
+	IsSuccessful := writerXml(config, configPathDir)
```

Comment:
watch your case. Use lowercase first letter for local variables only.

use uppercase for public functions only. If a function only lives in 1 file, leave the first letter as lowercase